### PR TITLE
assorted updates and fixes.

### PR DESCRIPTION
**Pull request by shlomif**

### CODE REVIEW

File: site/new_site_sources/templates/about_content.tmpl [hunk 1]
```diff
@@ -167,14 +167,17 @@ href="http://www.hamakor.org.il/%D7%A7%D7%94%D7%99%D7%9C%D7%94/%D7%99%D7%95%D7%9
...
-	 <a href="http://html-template.sourceforge.net/">HTML::Template</a> based site,
+	 <a href="https://metacpan.org/release/HTML-Template">HTML::Template</a> based site,
 	 with the look being based on
 	 the <a href="http://www.oswd.org/viewdesign.phtml?id=1192">Gila</a> template, which was
 	 downloaded from the <a href="http://www.oswd.org/">Open Source Web Design</a> site.<br />
-	 The site sources are managed under <a href="http://subversion.tigris.org/">Subversion</a>,
+	 The site sources are managed using <a href="https://github.com/szabgab/perl.org.il">a GitHub repository</a> - see <a href="https://guides.github.com/">their tutorials</a> for how to contribute.
+     </p>
+
+     <tmpl_if removed__subversion>
```
Comment (szabgab) [inline site/new_site_sources/templates/about_content.tmpl]: Why do we need this removed__subversion param? We are done with subversion we won't go back. You can remove all the relevant info. (Maybe keep a link to "historic" repo")

**Comment by shlomif** [comment]
On Thu, 19 Apr 2018 06:36:25 -0700
Gabor Szabo <notifications@github.com> wrote:

> szabgab commented on this pull request.
> 
> 
> 
> >  	 with the look being based on  
>  	 the <a href="http://www.oswd.org/viewdesign.phtml?id=1192">Gila</a>
> template, which was downloaded from the <a href="http://www.oswd.org/">Open
> Source Web Design</a> site.<br />
> -	 The site sources are managed under <a
> href="http://subversion.tigris.org/">Subversion</a>,
> +	 The site sources are managed using <a
> href="https://github.com/szabgab/perl.org.il">a GitHub repository</a> - see
> <a href="https://guides.github.com/">their tutorials</a> for how to
> contribute.
> +     </p>
> +
> +     <tmpl_if removed__subversion>
> 
> Why do we need this removed__subversion param? We are done with subversion we
> won't go back. You can remove all the relevant info. (Maybe keep a link to
> "historic" repo")
> 

I intended it to be similar to "#if 0" in C. But I can remove it completely.


-- 
-----------------------------------------------------------------
Shlomi Fish       http://www.shlomifish.org/
Let’s talk about restores instead of backups - http://is.gd/WatQqu

<rindolf> buu: do you have a functional spec? An architecture document? An
interface whitepaper? A developer’s guide? A user manual? A “The BL Book” and
“BL — The Program”?
<buu> rindolf: no, no, no no and no

Please reply to list if it's a mailing list post - http://shlom.in/reply .

**Comment by shlomif** [comment]
Done.

**Comment by szabgab** [comment]
thanks

**Comment by shlomif** [comment]
On Thu, 19 Apr 2018 08:21:36 -0700
Gabor Szabo <notifications@github.com> wrote:

> Merged #4.
> 

thanks!

-- 
-----------------------------------------------------------------
Shlomi Fish       http://www.shlomifish.org/
List of Networking Clients - http://shlom.in/net-clients

Jack: Hi Sophie!
Sophie: Don’t “Hi, Sophie!” me.
Jack: Don’t “Don’t ‘Hi, Sophie!’ me” me!

Please reply to list if it's a mailing list post - http://shlom.in/reply .

**Pull request merged**